### PR TITLE
Create break.md

### PR DESCRIPTION
**Pull request by PiaTio**
Hi!

### CODE REVIEW

**Comment by AHStoevenbelt** [comment]
:+1:

File: break.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Take a break
+
+After finishing a task, especially if it's a project, you need to take a break.
```
Comment (AHStoevenbelt) [inline break.md:3]: Totally agree, good point!

**Comment by PiaTio** [comment]
Fuck this shit.

2018-03-09 11:51 GMT+01:00 AHStoevenbelt <notifications@github.com>:

**Comment by AHStoevenbelt** [comment]
:D

At least there is cake.

**Pull request merged**